packages/lighthouse-service/src/property-manager/helpers.ts

```diff
@@ -0,0 +1,48 @@
+import fetch, { Headers } from "node-fetch";
+import { PropertyModel } from "./schema";
+
+export const getUserProfile = async (
+  rhUUID: string
+): Promise<PropertyUserProfileType> => {
+  const userQuery = `query GetUsers{
+      getUsersBy(rhatUUID:"${rhUUID}") {
+          name
```

Comment:
Use `cn` instead of the name as `name` is deprecated.

Reply:
```suggestion
          cn
```